Example/Podfile

```diff
@@ -1,4 +1,7 @@
 use_frameworks!
+
```

Comment:
Why 12? iirc we had to drop support for < 11?

Reply:
The latest Google IMA SDK requires a min deploy of 12, so I updated the lib to follow.